changelog/add-4017-enable-documents-feature

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Add documents and VAT invoices feature for supported countries
```

Comment:
Nitpick: Add a period at the end of the sentence. 😄

Reply:
Good catch! Fixed in https://github.com/Automattic/woocommerce-payments/pull/4199/commits/7f8376c130b5a80286a282b85a495d195b6480ea. I also noticed another changelog for the documents link was incorrect and fixed it in https://github.com/Automattic/woocommerce-payments/pull/4199/commits/c4a33dcc34cb4bac9ea72741502490c235315473.